app/dashboard/page.tsx

```
@@ -221,6 +226,8 @@ export default function DashboardPage() {
 
               <div className="grid grid-cols-3 gap-4 text-center mb-6">
                 <div>
+
+// TODO: Write function to get the number of items sold, active listings, and saved items
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)